.gitignore

```diff
@@ -1 +1,2 @@
 target
+.idea/
```

Comment:
Global ignore on your machine? NBD `:)`

Reply:
Ah yeah, I should do that. I'll remove it from here